site/public/css/electronic.css

```diff
@@ -1298,7 +1298,7 @@ tr.is_publish {
 .layout-option-cont {
     display: grid;
     grid-template-columns: repeat(auto-fill, 250px);
-    width: 83vw;
+    width: 78vw;
```

Comment:
Is there any reason this changed?

Reply:
After changing the popup to extend `Popup.twig`, it changed it slightly so if the width was too long, it would result in the inner container being larger it needed to be (the table was the correct size, but there was a horizontal drag bar that went out a bit to nothing). I tested it on Brave (Chromium, Windows) and Safari (iOS).